roles/sidekiq_worker/molecule/default/verify.yml

```diff
@@ -3,18 +3,16 @@
   hosts: all
   gather_facts: false
   tasks:
-  - name: check ruby package status
-    package:
-      name: "{{ item }}"
+  - name: test for deploy user on sudoers
+    lineinfile:
+      path: "/etc/sudoers"
+      line: "{{ item }}"
       state: present
-    check_mode: true
-    register: pkg_status
     loop:
-      - ruby-switch
-      - ruby2.6-dev
-      - ruby2.6
-
-  - name: test for ruby packages
-    assert:
-      that:
-        - not pkg_status.changed
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers status"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers restart"
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service sidekiq-workers start"
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
```

Comment:
The old test used to check for `f = host.file('/etc/systemd/system/sidekiq-workers.service')` Is this still needed?